bin/conpot

```diff
@@ -69,7 +69,7 @@ def logo():
 
 
 def on_unhandled_greenlet_exception(dead_greenlet):
-    logger.error('Stopping because %s died: %s', dead_greenlet, dead_greenlet.exception)
+    logger.exception('Stopping because %s died: %s', dead_greenlet, dead_greenlet.exception)
```

Comment:
Great start! Might as well [improve on string formatting](https://pyformat.info/) :slightly_smiling_face: 
```suggestion
    logger.exception('Stopping because {} died: {}'.format(dead_greenlet, dead_greenlet.exception))
```